docs/use/affinity.md

```diff
@@ -0,0 +1,170 @@
+|**Operator**|The operator field specifies how a label's values match a resource, such as a Pod or node. It establishes the logical condition to determine if a resource satisfies the affinity or anti-affinity rule.</br></br>The following are all logical operators you can use in the operator field:</br><ul><li>**In**: Matches label values in a list</li><li>**NotIn**: Matches label values not in a list.</li><li>**Exists**: Matches when a label key exists, regardless of its value.</li><li>**DoesNotExist**: Matches when a label key does not exist.</li></ul>|When using the **In** and **NotIn** operators, you have to provide the values for the key as well.|
+|**Values**|The values are the specific label values that must match the key for the affinity rule to apply.|Here are some examples for **values**:</br><ul><li>s2</li><li>database</li><li>production</li><li>custom values</li></ul>|
+
+## Use cases for affinity
+
+Here are several detailed use cases for affinity that highlight its diverse applications and the numerous benefits it offers across various contexts.
+
+
+=== "Node affinity"
+    ### Regional scheduling
+
+    Let's consider a use case in which workloads are distributed based on performance requirements, fault tolerance, and regional specifications across designated zones or areas.
+
+    You need to run a workload in the `us-west2 region` for latency optimaztion and to meet specific compliance requirements.
+
+    ```sh
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+            - key: topology.kubernetes.io/region
+            operator: In
+            values:
+            - "us-west2"
+    ```
+    ??? info "What happens under the hood"
+        - It ensures that the pod is scheduled only on nodes located in the us-west-2 region, as defined by the `topology.kubernetes.io/region` node label.
+
+        - For the rule to be applicable, it is essential that the node possesses a label such as `topology.kubernetes.io/region: us-west-2`. If no nodes correspond to the specified label, the pod will remain in an unscheduled state.
+
+
+=== "Pods affinity"
+    ### Pods scheduled together
+
+    Let's consider a use case in which you want to to ensure that HAProxy pods should be scheduled to run on the same Kubernetes node.
+    
+        affinity:
+          podAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+            matchExpressions:
+                - key: app
+                operator: In
+                values:
+                - haProxy
+        topologyKey: "kubernetes.io/hostname"
+
+    ??? info "What happens under the hood"
+        - The pod will be scheduled only on nodes that already contain pods labeled with `app=haproxy`.
+
+        - If no nodes match, the pod will not be scheduled until a suitable node becomes available.
```

Comment:
I don't think this a valid use case :confused: 
I think it's better to go with just 2 use cases that make a lot of sense than trying to squeeze in a 3rd case that doesn't just to show Pod Affinity.
```suggestion
```